pygpu/gpuarray.pyx

```diff
@@ -2229,13 +2229,40 @@ cdef class GpuArray:
         def __get__(self):
             return self.ga.offset
 
+    property data:
+        """Return a pointer to the raw OpenCL buffer object.
+
+        This will fail for arrays that have an offset.
```

Comment:
Add a comment that we support pyopencl interface that raise an error if offset isn't 0. It took me some times to understand why you chosed that interface instead of just adding the offset as gpudata